components/admin_console/billing/plan_details/plan_details.tsx

```diff
@@ -291,14 +301,14 @@ export const featureList = (isFreeTrial: boolean, isPaidTier: boolean) => {
         localizeMessage('admin.billing.subscription.planDetails.features.unlimitedIntegrations', 'Unlimited Integrations'),
     ];
 
-    // const featuresCloudStarter = [
-    //     localizeMessage('admin.billing.subscription.planDetails.features.groupAndOneToOneMessaging', 'Group and one-to-one messaging, file sharing, and search'),
-    //     localizeMessage('admin.billing.subscription.planDetails.features.incidentCollaboration', 'Incident collaboration'),
-    //     localizeMessage('admin.billing.subscription.planDetails.features.unlimittedUsersAndMessagingHistory', 'Unlimited users & message history'),
-    //     localizeMessage('admin.billing.subscription.planDetails.features.unlimitedIntegrations', 'Unlimited Integrations'),
-    //     localizeMessage('admin.billing.subscription.planDetails.features.mfa', 'Multi-Factor Authentication (MFA)'),
-    //     localizeMessage('admin.billing.subscription.planDetails.features.multilanguage', 'Multi-language translations'),
-    // ];
+    const featuresCloudStarter = [
+        localizeMessage('admin.billing.subscription.planDetails.features.groupAndOneToOneMessaging', 'Group and one-to-one messaging, file sharing, and search'),
+        localizeMessage('admin.billing.subscription.planDetails.features.incidentCollaboration', 'Incident collaboration'),
+        localizeMessage('admin.billing.subscription.planDetails.features.unlimittedUsersAndMessagingHistory', 'Unlimited users & message history'),
+        localizeMessage('admin.billing.subscription.planDetails.features.unlimitedIntegrations', 'Unlimited Integrations'),
```

Comment:
Katie mentioned that it is not clear if they will offer "Unlimited Integrations" so we should remove this from the list for now